superset/db_engine_specs.py

```diff
@@ -804,6 +809,21 @@ class PrestoEngineSpec(BaseEngineSpec):
             date_add('day', 1, CAST({col} AS TIMESTAMP))))",
     }
 
+    type_map = {
```

Comment:
Could this name be made a bit more self-documenting as to what it is mapping to what? (presto_to_?? ) If presto is the keys, what are the values? (upon first glance it looks like sqlalchemy abstract types, but then I see that mysql value there...)

Reply:
Ah yes. This is a mapping copied from the pyhive package (package we use for most of our presto needs). It's basically a mapping of strings to SQL object types defined by sqlalchemy. There isn't a tinyint data type defined by sqlalchemy, which is why we use mysql. But I agree it seems leaky. I can create a skeleton for tinyint and should create one for ROW object. From the looks of it, it looks like we just use its string representation in our code base so far.

Reply:
Created skeleton types and moved types to its own file.